### ISSUE TITLE
Millions of new materials discovered with deep learning

### ISSUE THREAD

**Issue by guevara** [guevara/read-it-later#10324]
Millions of new materials discovered with deep learning<br>
<br>
https://ift.tt/gVJ9oY5<br>
<br>
Amil Merchant and Ekin Dogus Cubuk<br>
<br>
<div><div>
<article>
<div>
<div>
<div>
<p>Research</p>
<dl>
<dt>Published</dt>
<dd>
<time datetime="2023-11-29">
29 November 2023
</time>
</dd>
<dt>Authors</dt>
</dl>
</div>

</div>
<div>
<p data-block-key="ulgua">AI tool GNoME finds 2.2 million new crystals, including 380,000 stable materials that could power future technologies</p>
<p data-block-key="3t80m">Modern technologies from computer chips and batteries to solar panels rely on inorganic crystals. To enable new technologies, crystals must be stable otherwise they can decompose, and behind each new, stable crystal can be months of painstaking experimentation.</p>
<p data-block-key="ofmi">Today, in a <a rel="noopener" href="https://www.nature.com/articles/s41586-023-06735-9" target="_blank">paper published in <i>Nature</i></a>, we share the discovery of 2.2 million new crystals – equivalent to nearly 800 years’ worth of knowledge. We introduce Graph Networks for Materials Exploration (GNoME), our new deep learning tool that dramatically increases the speed and efficiency of discovery by predicting the stability of new materials.</p>
<p data-block-key="ak633">With GNoME, we’ve multiplied the number of technologically viable materials known to humanity. Of its 2.2 million predictions, 380,000 are the most stable, making them promising candidates for experimental synthesis. Among these candidates are materials that have the potential to develop future transformative technologies ranging from superconductors, powering supercomputers, and next-generation batteries to boost the efficiency of electric vehicles.</p>
<p data-block-key="4too3">GNoME shows the potential of using AI to discover and develop new materials at scale. External researchers in labs around the world have independently created 736 of these new structures experimentally in concurrent work. In partnership with Google DeepMind, a team of researchers at the Lawrence Berkeley National Laboratory has also published <a rel="noopener" href="https://www.nature.com/articles/s41586-023-06734-w" target="_blank">a second paper in<i> Nature</i></a> that shows how our AI predictions can be leveraged for autonomous material synthesis.</p>
<p data-block-key="9fvee">We’ve made <a rel="noopener" href="https://www.nature.com/articles/s41586-023-06735-9" target="_blank">GNoME’s predictions available</a> to the research community. We will be contributing 380,000 materials that we predict to be stable to the Materials Project, which is now processing the compounds and adding them into <a rel="noopener" href="https://next-gen.materialsproject.org/" target="_blank">its online database</a>. We hope these resources will drive forward research into inorganic crystals, and unlock the promise of machine learning tools as guides for experimentation</p>
</div>
<p>
</p>
<h2 data-block-key="ulgua">Accelerating materials discovery with AI</h2>

<figure>
<figcaption>
<p data-block-key="6hdcv">About 20,000 of the crystals experimentally identified in the ICSD database are computationally stable. Computational approaches drawing from the Materials Project, Open Quantum Materials Database and WBM database boosted this number to 48,000 stable crystals. GNoME expands the number of stable materials known to humanity to 421,000.</p>
</figcaption>
</figure>
<div>
<p data-block-key="ulgua">In the past, scientists searched for novel crystal structures by tweaking known crystals or experimenting with new combinations of elements - an expensive, trial-and-error process that could take months to deliver even limited results. Over the last decade, computational approaches led by <a rel="noopener" href="https://materialsproject.org/" target="_blank">the Materials Project</a> and other groups have helped discover 28,000 new materials. But up until now, new AI-guided approaches hit a fundamental limit in their ability to accurately predict materials that could be experimentally viable. GNoME’s discovery of 2.2 million materials would be equivalent to about 800 years’ worth of knowledge and demonstrates an unprecedented scale and level of accuracy in predictions.</p>
<p data-block-key="c4h6v">For example, 52,000 new layered compounds similar to graphene that have the potential to revolutionize electronics with the development of superconductors. Previously, about <a rel="noopener" href="https://pubmed.ncbi.nlm.nih.gov/28191965/" target="_blank">1,000 such materials had been identified</a>. We also found 528 potential lithium ion conductors, 25 times more than a <a rel="noopener" href="https://pubs.rsc.org/en/content/articlelanding/2017/ee/c6ee02697d" target="_blank">previous study</a>, which could be used to improve the performance of rechargeable batteries.</p>
<p data-block-key="bh153">We are releasing the predicted structures for 380,000 materials that have the highest chance of successfully being made in the lab and being used in viable applications. For a material to be considered stable, it must not decompose into similar compositions with lower energy. For example, carbon in a graphene-like structure is stable compared to carbon in diamonds. Mathematically, these materials lie on the convex hull. This project discovered 2.2 million new crystals that are stable by current scientific standards and lie below the convex hull of previous discoveries. Of these, 380,000 are considered the most stable, and lie on the “final” convex hull – the new standard we have set for materials stability.<br></p>
<h2 data-block-key="a60kf">GNoME: Harnessing graph networks for materials exploration</h2>
</div>
<figure>
<figcaption>
<p data-block-key="6hdcv">GNoME uses two pipelines to discover low-energy (stable) materials. The structural pipeline creates candidates with structures similar to known crystals, while the compositional pipeline follows a more randomized approach based on chemical formulas. The outputs of both pipelines are evaluated using established Density Functional Theory calculations and those results are added to the GNoME database, informing the next round of active learning.</p>
</figcaption>
</figure>
<div>
<p data-block-key="ulgua">GNoME is a state-of-the-art graph neural network (GNN) model. The input data for GNNs take the form of a graph that can be likened to connections between atoms, which makes GNNs particularly suited to discovering new crystalline materials.</p>
<p data-block-key="dn385">GNoME was originally trained with data on crystal structures and their stability, openly available through <a rel="noopener" href="https://next-gen.materialsproject.org/" target="_blank">the Materials Project</a>. We used GNoME to generate novel candidate crystals, and also to predict their stability. To assess our model’s predictive power during progressive training cycles, we repeatedly checked its performance using established computational techniques known as Density Functional Theory (DFT), used in physics, chemistry and materials science to understand structures of atoms, which is important to assess the stability of crystals.</p>
<p data-block-key="1v2m6">We used a training process called ‘active learning’ that dramatically boosted GNoME’s performance. GNoME would generate predictions for the structures of novel, stable crystals, which were then tested using DFT. The resulting high-quality training data was then fed back into our model training.</p>
<p data-block-key="3ja0j">Our research boosted the discovery rate of materials stability prediction from around 50%, to 80% - based on an external benchmark set by previous state-of-the-art models. We also managed to scale up the efficiency of our model by improving the discovery rate from under 10% to over 80% - such efficiency increases could have significant impact on how much compute is required per discovery.</p>
<h2 data-block-key="9q825">AI ‘recipes’ for new materials</h2>
<p data-block-key="ei4a6">The GNoME project aims to drive down the cost of discovering new materials. External researchers have independently created 736 of GNoME’s new materials in the lab, demonstrating that our model’s predictions of stable crystals accurately reflect reality. We’ve released our database of newly discovered crystals to the research community. By giving scientists the full catalog of the promising ‘recipes’ for new candidate materials, we hope this helps them to test and potentially make the best ones.</p>
</div>
<figure>
<figcaption>
<p data-block-key="sfacz">Upon completion of our latest discovery efforts, we searched the scientific literature and found 736 of our computational discoveries were independently realized by external teams across the globe. Above are six examples ranging from a first-of-its-kind Alkaline-Earth Diamond-Like optical material (Li4MgGe2S7) to a potential superconductor (Mo5GeB2).</p>
</figcaption>
</figure>
<p data-block-key="ulgua">Rapidly developing new technologies based on these crystals will depend on the ability to manufacture them. In a paper led by our collaborators at Berkeley Lab, researchers showed a robotic lab could rapidly make new materials with automated synthesis techniques. Using materials from the Materials Project and insights on stability from GNoME, the autonomous lab created new recipes for crystal structures and successfully synthesized more than 41 new materials, opening up new possibilities for AI-driven materials synthesis.</p>
<figure>
<figcaption>
<p data-block-key="bu4rv">A-Lab, a facility at Berkeley Lab where artificial intelligence guides robots in making new materials. Photo credit: Marilyn Sargent/Berkeley Lab</p>
</figcaption>
</figure>
<div>
<h2 data-block-key="2wk2x">New materials for new technologies</h2>
<p data-block-key="5lhj1">To build a more sustainable future, we need new materials. GNoME has discovered 380,000 stable crystals that hold the potential to develop greener technologies – from better batteries for electric cars, to superconductors for more efficient computing.</p>
<p data-block-key="80bof">Our research – and that of collaborators at the Berkeley Lab, Google Research, and teams around the world — shows the potential to use AI to guide materials discovery, experimentation, and synthesis. We hope that GNoME together with other AI tools can help revolutionize materials discovery today and shape the future of the field.</p>
</div>
<aside>
<h2>Read our paper in Nature</h2>
<ul>
<li><a rel="noopener" data-gtm-tag="cta-selection" href="https://www.nature.com/articles/s41586-023-06735-9" target="_blank">
Read our paper in Nature

</a></li>
<li><a rel="noopener" data-gtm-tag="cta-selection" href="https://www.nature.com/articles/s41586-023-06734-w" target="_blank">
Read the Berkeley Lab paper in Nature

</a></li>
<li><a rel="noopener" data-gtm-tag="cta-selection" href="https://github.com/google-deepmind/materials_discovery" target="_blank">
Download the dataset

</a></li>
</ul>
</aside>
<aside>
<div>
<div>
<h2 data-block-key="eggvz">Acknowledgements</h2>
<p data-block-key="9jjr8">This work would not have been possible without our amazing co-authors: Simon Batzner, Sam Schoenholz, Muratahan Aykol, and Gowoon Cheon. We would also like to acknowledge Doug Eck, Jascha Sohl-dickstein, Jeff Dean, Joëlle Barral, Jon Shlens, Pushmeet Kohli, and Zoubin Ghahramani for sponsoring the project; Lizzie Dorfman for Product Management support; Andrew Pierson for Program Management support; Ousmane Loum for help with computing resources; Luke Metz for his help with infrastructure; Ernesto Ocampo for help with early work on the AIRSS pipeline; Austin Sendek, Bilge Yildiz, Chi Chen, Chris Bartel, Gerbrand Ceder, Joy Sun, JP Holt, Kristin Persson, Lusann Yang, Matt Horton, and Michael Brenner for insightful discussions; and the Google DeepMind team for continuing support.</p>
</div>
</div>
</aside>
</div>
</article>
</div></div><br>
<br>
<br>
<br>
<br>
<br>
via Google DeepMind https://deepmind.google<br>
<br>
December 1, 2023 at 01:42PM
